doc/news/changes/incompatibilities/20170810MatthiasMaier

```diff
@@ -1,8 +1,14 @@
-Removed: LogStream: The reproducible test functionality has been removed
-from deallog. This includes <code>test_mode()</code>,
-<code>threshold_double()</code>, and <code>threshold_float()</code>. Use
-numdiff, or other tools that can do output comparison with small numerical
-differences instead. Further, the <code>log_cerr()</code> function has been
-removed.
+Removed: A number of old functionality has been removed from LogStream:
```

Comment:
"old functionality" is not countable. -> "Some old functionality" or "A significant amount of old functionality"